Gemfile

```diff
@@ -8,6 +8,10 @@ gem 'rails', '4.1.9'
 # use postgres database
 gem 'pg'
 
+group :development, :test do
+  gem 'sqlite3'
```

Comment:
@gschorkopf I didn't have postgres setup when @camthompson and I initially worked on this so we used sqlite as a workaround. Will remove